src/lib/components/embla/index.ts

```diff
@@ -0,0 +1,7 @@
+import EmblaRoot from './Embla.svelte';
+import EmblaSlide from './EmblaSlide.svelte';
+
+export const Embla = Object.assign({
+    Root: EmblaRoot,
+    Slide: EmblaSlide
+});
```

Comment:
```suggestion
export { default as Root } from './Embla.svelte';
export { default as Slide } from './EmblaSlide.svelte';
```

does this work? 👀

Reply:
It does, just requires multiple imports. Easy change!